db/schema.rb

```diff
@@ -132,6 +132,7 @@
     t.string   "cove_uri"
     t.string   "origin"
     t.boolean  "vetted"
+    t.string   "rep_group"
```

Comment:
This should not be necessary, as it should be handled by the `acts_as_taggable_on` gem